plugins/arDominionB5Plugin/modules/repository/templates/_holdingsInstitution.php

```diff
@@ -1,38 +1,24 @@
-<section id="repo-holdings" class="list-menu"
-  data-total-pages="<?php echo $pager->getLastPage(); ?>"
-  data-url="<?php echo url_for(['module' => 'repository', 'action' => 'holdingsInstitution', 'id' => $resource->id]); ?>">
+<section class="card mb-4">
 
-  <div class="panel panel-gray">
-    <div class="panel-body">
+  <div class="p-4 pb-0">
+    <?php include_component('repository', 'logo'); ?>
+  </div>
 
-      <?php use_helper('Text'); ?>
+  <div class="card-body">
 
-      <div class="repository-logo<?php echo $resource->existsLogo() ? '' : ' repository-logo-text'; ?>">
-        <a href="<?php echo url_for([$resource, 'module' => 'repository']); ?>">
-          <?php if ($resource->existsLogo()) { ?>
-            <?php echo image_tag($resource->getLogoPath(),
-                  ['alt' => __('Go to %1%',
-                  ['%1%' => truncate_text(strip_markdown($resource), 100)])]); ?>
-          <?php } else { ?>
-            <h2><?php echo render_title($resource); ?></h2>
-          <?php } ?>
-        </a>
+    <form class="mb-4" role="search" aria-label="<?php echo sfConfig::get('app_ui_label_institutionSearchHoldings'); ?>" action="<?php echo url_for(['module' => 'informationobject', 'action' => 'browse']); ?>">
+      <input type="hidden" name="repos" value="<?php echo $resource->id; ?>">
+      <label for="institution-search-query" class="h5 mb-2 form-label"><?php echo sfConfig::get('app_ui_label_institutionSearchHoldings'); ?></label>
+      <div class="input-group">
+        <input type="text" class="form-control" id="institution-search-query" name="query" value="<?php echo $sf_request->query; ?>" placeholder="<?php echo __('Search'); ?>" required>
+        <button class="btn atom-btn-white" type="submit" aria-label=<?php echo __('Search'); ?>>
+          <i aria-hidden="true" class="fas fa-search"></i>
+        </button>
       </div>
+    </form>
 
-      <h3><?php echo sfConfig::get('app_ui_label_institutionSearchHoldings'); ?></h3>
-
-      <form class="mb-3" role="search" aria-label="<?php echo sfConfig::get('app_ui_label_institutionSearchHoldings'); ?>" action="<?php echo url_for(['module' => 'informationobject', 'action' => 'browse']); ?>">
-        <input type="hidden" name="repos" value="<?php echo $resource->id; ?>">
-        <div class="input-group input-group-sm">
-          <input type="text" class="form-control" name="query" aria-label="<?php echo __('Search'); ?>" value="<?php echo $sf_request->query; ?>" placeholder="<?php echo __('Search'); ?>">
-          <button class="btn atom-btn-white" type="submit" aria-label=<?php echo __('Search'); ?>>
-            <i aria-hidden="true" class="fas fa-search"></i>
-          </button>
-        </div>
-      </form>
+    <?php echo get_component('menu', 'browseMenuInstitution', ['sf_cache_key' => $sf_user->getCulture().$sf_user->getUserID()]); ?>
```

Comment:
Oh, can you use a different key for the B5 theme? Similar to https://github.com/artefactual/atom/blob/dev-test/issue-13530-b5-themes/plugins/arDominionB5Plugin/templates/_header.php#L29. Not a big deal but it helps with the `theme` parameter added for testing, and it may help in the future to improve theme selection.

Reply:
Fixed.